keeper/src/main/scala/zio/keeper/errors.scala

```diff
@@ -4,7 +4,7 @@ import zio.duration.Duration
 import zio.keeper.transport.Address
 import zio.nio.core.SocketAddress
 
-sealed abstract class Error(val msg: String = "") {
+sealed abstract class Error(val msg: String = "") extends Throwable {
```

Comment:
I'd extend `Exception` (or `RuntimeException`). Not sure whether or not it make sense to disable stack trace accumulation, e.g. `extends Exception(msg, null, false, false)`.

Reply:
@mschuwalow what do you think ?

Reply:
I'm personally a fan of keeping stack traces. Main reason for dropping them would be performance, no? In that case let's keep them for now and rip them out when we do a performance pass

Reply:
so I will change that to `Exception` and keep stacktrace for the time being.

Reply:
done